.travis.yml

```diff
@@ -8,30 +8,29 @@ cache:
 
 env:
   global:
-    - MODE="test"
     - NUMPY="numpy"
     - SCIPY="false"
     - COVERAGE="false"
+    - PYTHON="3.6"
 
 jobs:
   include:
     - stage: basic
-      env: MODE="static" PYTHON="2.7"
-    - env: MODE="test" PYTHON="3.6" COVERAGE="true" SCIPY="true"
-    - env: MODE="test" PYTHON="2.7" COVERAGE="true"
+      env: MODE="static"
+    - env: MODE="test" COVERAGE="true" SCIPY="true"
```

Comment:
The fact we're no longer collecting coverage on 2.7 seems to be changing things quite a bit. Should I be going through the codecov report in detail and fixing up all the missing coverage?